src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,27 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  gap: 48px;
+  justify-content: center;
+
+  $media-querry:
+    2 488px,
+    3 768px,
+    4 1024px;
```

good implementation 🔥 but you also can include your default template to this loop
```suggestion
  display: grid;
  gap: 48px;
  justify-content: center;

  $media-querry:
  	1 0
    2 488px,
    3 768px,
    4 1024px;
```